backends/edxorg_test.py

```diff
@@ -0,0 +1,51 @@
+"""
+Oauth Backend Tests
+"""
+# pylint: disable=no-self-use
+from unittest import TestCase
+from .edxorg import EdxOrgOAuth2
+
+
+class EdxOrgOAuth2Tests(TestCase):
+    """Tests for EdxOrgOAuth2"""
+    def test_response_parsing(self):
+        """
+        Should have properly formed payload if working.
+        """
+        eoo = EdxOrgOAuth2()
+        result = eoo.get_user_details({
+            'id': 5,
+            'username': 'darth',
+            'email': 'darth@deathst.ar',
+            'name': 'Darth Vader'
+        })
+
+        assert {
+            'edx_id': 5,
+            'username': 'darth',
+            'fullname': 'Darth Vader',
+            'email': 'darth@deathst.ar',
+            'first_name': 'Darth',
+            'last_name': 'Vader'
+        } == result
+
+    def test_single_name(self):
+        """
+        If the user only has one name, last_name should be blank.
+        """
+        eoo = EdxOrgOAuth2()
+        result = eoo.get_user_details({
+            'id': 5,
+            'username': 'staff',
+            'email': 'staff@example.com',
+            'name': 'staff'
+        })
+
+        assert {
+            'edx_id': 5,
+            'username': 'staff',
+            'fullname': 'staff',
+            'email': 'staff@example.com',
+            'first_name': 'staff',
+            'last_name': ''
+        } == result
```

Comment:
https://github.com/omab/python-social-auth/blob/master/social/backends/base.py#L167

Handled. You think I would forget about you?